src/__tests__/index.ts

```diff
@@ -61,6 +62,23 @@ describe('LinkSDK', () => {
       expect(mockValue.oauthParams.scope).toBe(scope.join(' '));
     });
 
+    test('invalid scope', async () => {
+      // @ts-ignore Ignores invalid scope to test its usage
+      const scope: IScope[] = [value];
+
+      try {
+        linkSDK.init.call(mockValue, {
+          clientId: value,
+          scope
+        });
+
+        throw new Error('failed');
```

Comment:
you can try to use this for all throw exception
https://jestjs.io/docs/en/expect#tothrowerror